net/nimble/host/services/dis/syscfg.yml

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Package: ble/host/services/dis
+
+syscfg.defs:
+    BLE_SVC_DIS_MODEL_NUMBER_READ_PERM:
+        description: >
+            Defines permissions for writing "Model Number" characteristics. Can
```

Comment:
Same thing here, these should all be Defines permissions for *reading

Reply:
Also looking at this now I wonder if we should have just one read_perm syscfg for all characteristics and enable for each characteristic separately? I am not sure if there is scenario for setting different permissions for each characteristic here.

Not really sure what is the proper solution here, just thinking out loud.

Reply:
Perhaps a `BLE_SVC_DIS_DEFAULT_READ_PERM`?
And propose the `MYNEWT_VAL_BLE_SVC_DIS_DEFAULT_READ_PERM` as a possible alternative for the `BLE_GATT_CHR_F_READ_*` values in `BLE_SVC_DIS_*_READ_PERM`